openshift/keycloak.app.yaml

```diff
@@ -13,6 +13,13 @@ objects:
   spec:
     strategy:
       type: Rolling
+      resources:
+        requests:
+          cpu: 50m
+          memory: 500Mi
+        limits:
+          cpu: 500m
+          memory: 2Gi
```

Comment:
Not sure about the CPU. For memory it should be fine. As commented by Hector in the DB part - KC is quite a beast. The thing which makes me wonder though is why there is no GC affecting mem usage. I think as next step we should try to monitor jvm heap for each pod and tweak jvm startup settings.